src/components/App/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,40 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`App has valid snapshot 1`] = `
```

Comment:
Need to find out if committing snapshots is fine, or should the be ignored and generated on CI build.

Reply:
Add this to trello board as a spike. 👍